### PR TITLE
Mocked Base Learner Record Course Component

### PR DESCRIPTION
**Pull request by MatthewBlanski**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<template>
+    <template if:false={shouldDisplayCourses}>
+        <!-- Display spinner -->
+        <lightning-spinner alternative-text={labelReference.spinnerLoadingAltText} variant="brand" size="medium">
+        </lightning-spinner>
+    </template>
+    {jsonDebug}
```
Comment (salvadorsfdc) [inline force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html:7]: Maybe we don't need this anymore?

File: force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <lightning-card
+        icon-name="custom:custom83"
+        title={learnerCourseViewModel.schemaLabels.assertion.achievement.achievementType}
+        if:true={shouldDisplayCourses}
+    >
+        <ul class="slds-accordion">
+            <template for:each={learnerCourseViewModel.courses} for:item="learnerCourse">
```
Comment (salvadorsfdc) [inline force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html:14]: Do we need a message if there's no courses not display?

**Review by salvadorsfdc — CHANGES_REQUESTED** [review]
Looks good, just 2 questions

File: force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html (hunk 1 above)
Comment (MatthewBlanski) [reply force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html:7]: No we do not!

File: force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html (hunk 2 above)
Comment (MatthewBlanski) [reply force-app/main/universalLearner/lwc/learnerCourseVisualizer/learnerCourseVisualizer.html:14]: Right now, no. We may choose not to display the entire component at all, but that's TBD

**Review by salvadorsfdc — APPROVED** [review]
Looks good, approving changes.

**Pull request merged**